src/content-cli-export.ts

```diff
@@ -13,36 +14,47 @@ export class Export {
             .requiredOption("--packageKeys <packageKeys...>", "Exports only given package keys")
             .option("--includeDependencies", "Include variables and dependencies", "")
             .action(async cmd => {
-                await new PackageCommand().batchExportPackages(cmd.packageKeys, cmd.includeDependencies)
+                await new PackageCommand().batchExportPackages(cmd.packageKeys, cmd.includeDependencies);
                 process.exit();
             });
 
         return program;
     }
-}
 
-const options = commander.parseOptions(process.argv)
-const indexOfProfileOption = options.unknown.indexOf('-p') !== -1 ? options.unknown.indexOf('-p') : options.unknown.indexOf('--profile');
+    public static dataPool(program: CommanderStatic): CommanderStatic {
+        program
+            .command("data-pool")
+            .description("Command to export a data pool")
+            .option("-p, --profile <profile>", "Profile which you want to use to export the data pool")
+            .requiredOption("--id <id>", "ID of the data pool you want to export")
+            .option("--outputToJsonFile", "Output the exported data pool to a JSON file")
+            .action(async cmd => {
+                await new DataPoolCommand().exportDataPool(cmd.id, cmd.outputToJsonFile);
+                process.exit();
+            });
 
-process.on("unhandledRejection", (e, promise) => {
-    logger.error(e.toString());
-})
+        return program;
+    }
+}
 
-contextService.resolveProfile(options.unknown[indexOfProfileOption + 1]).then(() => {
-    getAllCommands();
-}, ()=> {
+const loadCommands = () => {
     getAllCommands();
-}).catch(e => {
-    console.log(e)
-});
+};
+
+ContextInitializer.initContext()
+    .then(loadCommands, loadCommands)
```

Comment:
Are we calling the loadCommands twice here? I haven't seen this syntax before/

Reply:
it's exactly as it was before, just instead of writting the () => {} twice, I put it into a variable.